benchexec/tools/divine4.py

```diff
@@ -114,7 +114,7 @@ def determine_result(self, returncode, returnsignal, output, isTimeout):
         (e.g., "CRASH", "OUT_OF_MEMORY", etc.).
         """
 
-        if not output:
+        if not output or output[-1] == None:
```

Comment:
This doesn't make sense: `output` is a list of strings, it will never contain `None`.
Furthermore, one cannot compare with `None` using `==` in Python ([explanation](https://stackoverflow.com/q/14247373/396730)).